test/functional/tools/mulled_example_multi_incomplete.xml

```diff
@@ -0,0 +1,19 @@
+<tool id="mulled_example_multi_incomplete" name="mulled_example_multi_incomplete" version="0.1.0">
+    <command><![CDATA[
+        bedtools --version > $out_file1 ;
+        echo "Moo" >> $out_file1 ;
+        samtools >> $out_file1 2>&1 ;
+        echo "Cow" >> $out_file1 ;
+    ]]></command>
+    <requirements>
+        <requirement type="package" version="1.3.1">samtools</requirement>
+        <requirement type="package" version="2.26.0">bedtools</requirement>
+        <requirement type="package" version="4.5">notarealpackage</requirement>
```

Comment:
I'm not sure I understand why notarealpackage is included. The offline bug seems to be independent of whether a dependency can be resolved, right?